web/client/api/WMS.js

```diff
@@ -84,12 +181,13 @@ export const getOnlineResource = (c) => {
     return c.Request && c.Request.GetMap && c.Request.GetMap.DCPType && c.Request.GetMap.DCPType.HTTP && c.Request.GetMap.DCPType.HTTP.Get && c.Request.GetMap.DCPType.HTTP.Get.OnlineResource && c.Request.GetMap.DCPType.HTTP.Get.OnlineResource.$ || undefined;
 };
 export const searchAndPaginate = (json = {}, startPosition, maxRecords, text) => {
```

Comment:
Only one reminder: 
These functions (`searchText` and `getRecords`) are not part of WMS protocol. 
I started to isolate specific protocols APIs (WMS, WMTS...) from catalog abstraction, but we have not finished yet. 
They will need sooner or later to be moved in `api/catalog/WMS`. Anyway in this PR we can highlight the changes on the specific task. So I should move them in a separate PR. 

Try only to keep isolated the 2 concepts. (WMS protocol vs Catalog abstraction).